shared/chat/conversation/info-panel/notifications/container.js

```diff
@@ -84,14 +90,14 @@ export default compose(
       },
       updateDesktop: (state, props) => (desktop: Types.NotificationsType) => {
         if (desktop === state.desktop) {
-          return {}
+          return undefined
```

Comment:
according to the recompose docs, returning undefined skips a re-layout

Reply:
i think {} also works but this is fine https://github.com/acdlite/recompose/blob/master/src/packages/recompose/withStateHandlers.js#L34

Reply:
can also just 'return' don't need to return undefined

Reply:
Ah, you're right, I had assume they special-cased `undefined`. I'll just revert back.

Reply:
nah lets leave it, just make it `return`

Reply:
Done